src/component/handlers/composition/DraftEditorCompositionHandler.js

```diff
@@ -207,35 +208,20 @@ const DraftEditorCompositionHandler = {
         currentStyle,
         entityKey,
       );
+
+      editor.registerComposedBlockKey(blockKey);
+
       // We need to update the editorState so the leaf node ranges are properly
       // updated and multiple mutations are correctly applied.
       editorState = EditorState.set(editorState, {
         currentContent: contentState,
       });
     });
 
-    // When we apply the text changes to the ContentState, the selection always
-    // goes to the end of the field, but it should just stay where it is
-    // after compositionEnd.
-    const documentSelection = getDraftEditorSelection(
-      editorState,
-      getContentEditableContainer(editor),
-    );
-    const compositionEndSelectionState = documentSelection.selectionState;
-
-    editor.restoreEditorDOM();
-
-    const editorStateWithUpdatedSelection = EditorState.acceptSelection(
-      editorState,
-      compositionEndSelectionState,
-    );
```

Comment:
@bchmn Lets verify that we don't need update the selection. Is there a chance selection would get overridden by changing the block key?

Reply:
verified, selection is fine